index.js

```diff
@@ -24,14 +24,21 @@ function Lash (spec) {
     var self = {};
     if (!spec) spec = {};
 
-    self.startTime = null;
+    self.timers = {};
 
-    self.startTimer = function () {
-        self.startTime = process.hrtime();
+    self.startTimer = function (timerId) {
+        if (!timerId) timerId = 'default';
```

Comment:
jut throw an error here? `if typeof timerId !== 'string' throw ...` sorta thing? That way we don't mistakenly forget to use an ID when collecting the timer and return two default times.